### PR TITLE
x230: improve the documentation of the different release images / con…

### LINKED ISSUE #56: Clarify the difference between the free and nonfree image
It would be great to clarify the difference between the free and nonfree image in the x230 README.
I.e.

- Do both images work (for everybody) ?
- If not, why ?
- What are the (expected/known) issue with the free image ?
- Is there anything that a user can do to use the free image is failing out of the box ?

### PR DESCRIPTION
**Pull request by merge**
…figs

Fixes #56

### CODE REVIEW

File: x230/README.md [hunk 1]
```diff
@@ -8,6 +8,21 @@ Get it from our [release page](https://github.com/merge/coreboot-x230/releases)
 * __microcode update__: revision `20` from 2018-04-10 (includes mitigations for Spectre Variant 3a and 4)
 * __SeaBIOS__: version [1.12.0](https://seabios.org/Releases) from 2018-11-17
 
+### release images to choose from
+We release multiple different, but _very similar_ images you can choose from.
+They all should work on all versions of the X230/X230T. These are the
+differences; xxxxxxxxxx is a placeholder for the unique ID in the filename:
```
Comment (varac) [inline x230/README.md]: "the unique ID" should be better explained ( idk what you mean with it )

File: x230/README.md [hunk 2]
```diff
@@ -8,6 +8,21 @@ Get it from our [release page](https://github.com/merge/coreboot-x230/releases)
...
+[VGA BIOS](https://en.wikipedia.org/wiki/Video_BIOS) from [Intel](https://www.intel.com/content/www/us/en/intelligent-systems/intel-embedded-graphics-drivers/faq-bios-firmware.html)
+which is non-free software. It is executed in "secure" mode.
+* `x230_coreboot_seabios_free_xxxxxxxxxx_top.rom` includes  the
+[VGA BIOS](https://en.wikipedia.org/wiki/Video_BIOS)
+[SeaVGABIOS](https://www.seabios.org/SeaVGABIOS) which is free software.
+While technically more interesting, visually this is currently not as
+beautiful:
+  * The bootspash image is not shown
+  * Early boot console messages (after your HDD's bootloader has started a kernel) might be [missing](https://github.com/merge/skulls/issues/46)
+
```
Comment (varac) [inline x230/README.md]: Please also add what image is used by default by the flash script.

Comment (merge) [reply x230/README.md]: there's no default. the `-i image_filename` option is documented if you want to automate things, and by default the scripts let you choose interactively.

Comment (varac) [reply x230/README.md]: Ok - then it would be nice to point the user to this section in the README to help making an educated choice.

**Pull request closed without merging**